### PR TITLE
Fixes autolathe "tinfoil hat" bug by renaming Misc section

### PR DESCRIPTION
**Pull request by ChesterTheCheesy**
# General Documentation

### Intent of your Pull Request
Fixes bug properly without nuking the poor hat. It's innocent.

### Why is this change good for the game?
bugs bad, tinfoil hat god

### Briefly describe your PR and the impacts of it, in layman's terms. 
renames "Misc" category to "Miscellaneous", as the "Misc" really screws up when there's other categories named similarly (Misc. Machinery), making the latter show up indefinitely, but with Tinfoil hat, as it's the last saved value in that case. Also moves the mounted flash frame design to correct file.

### What should players be aware of when it comes to the changes your PR is implementing?
TINFOIL HAT [x5]
TINFOIL HAT [x5]
TINFOIL HAT [x5]
isn't a case anymore
### What general grouping does this PR fall under? 
bugfix

# Changelog

:cl:  
bugfix: fixed autolathe bug, moved one protolathe only design to correct file
/:cl:

### CODE REVIEW

**Comment by ChesterTheCheesy** [comment]
However, i could be wrong on the source of the bug, as i don't know javascript that well.

**Review by jcatjmeow — APPROVED** [review]
